subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -311,6 +313,11 @@ int fota_download_init(fota_download_callback_t client_callback)
 
 	callback = client_callback;
 
+#ifdef CONFIG_DFU_TARGET_MCUBOOT
+	/* Set the required buffer for MCUboot targets */
+	(void)dfu_target_mcuboot_set_buf(mcuboot_buf, sizeof(mcuboot_buf));
```

Comment:
IMO we should check the error code here. Throwing away error codes like this does not look that good, also we do error checking below.